tests/playwright/sanity/container.test.js

```diff
@@ -43,6 +43,165 @@ test.describe( 'Container tests', () => {
 		} );
 	} );
 
+	test( 'Test widgets display inside the container using various directions and content width', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			containerId = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Close Navigator
+		await page.click( '#elementor-navigator__close' );
+
+		// Act.
+		// Add widgets.
+		await editor.addWidget( 'accordion', containerId );
+		await editor.addWidget( 'divider', containerId );
+		await editor.addWidget( 'spacer', containerId );
+		await editor.addWidget( 'toggle', containerId );
+		await editor.addWidget( 'video', containerId );
+
+		// Select spacer element.
+		const spacerElement = await editor.getPreviewFrame().waitForSelector( '.elementor-widget-spacer' );
+		await spacerElement.hover();
+		const spacerEditButton = await editor.getPreviewFrame().waitForSelector( '.elementor-widget-spacer .elementor-editor-element-edit' );
+		await spacerEditButton.click();
+		// Set background colour.
+		await page.locator( '.elementor-tab-control-advanced' ).click();
+		await page.locator( '.elementor-control-_section_background .elementor-panel-heading-title' ).click();
+		await page.locator( '.elementor-control-_background_background .eicon-paint-brush' ).click();
+		await page.locator( '.elementor-control-_background_color .pcr-button' ).click();
+		await page.locator( '.pcr-app.visible .pcr-interaction input.pcr-result' ).fill( '#A81830' );
+		// Select container.
+		const containerElement = await editor.getPreviewFrame().waitForSelector( '.elementor-element-' + containerId );
+		await containerElement.hover();
+		const containerEditButton = await editor.getPreviewFrame().waitForSelector( '.elementor-element-' + containerId + ' .elementor-editor-element-edit' );
+		await containerEditButton.click();
+		// Set row direction.
+		await page.click( '.elementor-control-flex_direction i.eicon-arrow-right' );
+
+		const container = editor.getPreviewFrame().locator( '.elementor-element-' + containerId );
```

Comment:
Don't you already have this from line 79? What is the difference between them?

Reply:
We make  2 screenshots: 
- position: absolute, content-width: full
- position: absolute, content-width: boxed

Reply:
We make 4 screenshots: 
- container, content-width: full, direction: row
- container, content-width: boxed, direction: row
- container, content-width: boxed, direction: column
- container, content-width: full, direction: column